src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -240,14 +242,24 @@ export function validateAddress(errors, formData) {
 export const download4142Notice = (
   <div className="usa-alert usa-alert-warning no-background-image">
     <p>
-      Since your doctor has your private medical records, you’ll need to fill
+      Since your doctor has your private medical records, you'll need to fill
       out an Authorization to Disclose Information to the VA (VA Form 21-4142) so
-      we can request your records. You’ll need to fill out a form for each doctor.
+      we can request your records.
     </p>
     <p>
-      <a href="https://www.vba.va.gov/pubs/forms/VBA-21-4142-ARE.pdf" target="_blank">
+      <a href={vaForm4142URL} target="_blank">
         Download VA Form 21-4142
       </a>.
+      <p>
+        Please print the form, fill it out, and send it to:
+      </p>
+      <p className="va-address-block">
```

Comment:
I believe this address is used in at least two places:
1. On this page's content
2. On the confirmation page's content

Would it be worth pulling the address `jsx` out into its own variable?